css/style.css

```diff
@@ -53,7 +53,7 @@ a:hover {
 
 /* picture sector styling */
```

Comment:
Hi Anna, really good job, well done! I learned from your project that we can use multiple CSS files for a website. Before I thought we can use only one for all pages)) Also, I've noticed you use "div" a lot,   we can use "section" or  "article" as well. I noticed it's easier to navigate through the code in the future (for me).